src/Services/InsuranceCompanyService.php

```diff
@@ -256,7 +257,8 @@ public function insert($data)
                 $data["ins_type_code"],
                 $data["x12_receiver_id"],
                 $data["x12_default_partner_id"] ?? '',
-                $data["alt_cms_id"]
+                $data["alt_cms_id"],
+                $data["cqm_sop"] ?? null,
```

Comment:
Looks like a new column. If so, Do I need to change import destination for payer. Also payer cqm service.

Reply:
okay. please remind me when merged.